### ISSUE TITLE
Drag and drop of notes not working because of BBT related scrypt in Zotero

### ISSUE THREAD

**Issue by betamigo98** [retorquere/zotero-better-bibtex#2155]
Hi and may thank for the work and the value you add with BBT,
In Zotero I cannot drag and drop childnote neither standalone notes.
Trying to on/off plugins one by one, it seems like BBT is involved.
I tried from zotero to libreoffice document and to a mail service.
More details and error report posted in zotero [here](https://forums.zotero.org/discussion/comment/408255/#Comment_408255)

Would you please take a look?
Best regards

**Comment by betamigo98** [comment]
Not related to reference supportlog ID :  D598714194

**Comment by retorquere** [comment]
And this is specific to bbt? Other translatiors do not have this problem?

**Comment by retorquere** [comment]
AFAICT you have chicago set as the D&D format? I'm not sure BBT should be involved at all. I've asked dstillman.

**Comment by betamigo98** [comment]
Mmm after a re-do it seems that Mdnote is the plugin involved.
...own you an apology

**Comment by betamigo98** [comment]
> AFAICT you have chicago set as the D&D format? I'm not sure BBT should be involved at all. I've asked dstillman.

I do have the Chicago style as default export format dor note. See picture below.
![image](https://user-images.githubusercontent.com/105124171/168680184-ba8d50e5-fe75-4404-8893-132008aa7abc.png)

**Comment by betamigo98** [comment]
Last update: Mdnote seems to generate the unexpected behaviour, but so do the BBT. So it is both of them.

**Comment by betamigo98** [comment]
Same process with APA7th generate this suport log ID KW9RYBTT-euc
I was trying to drag a note in this github space for comment.
Not in the log but I tried to drag and drop to libreoffice, same issue.
Picture of preferences below:
![image](https://user-images.githubusercontent.com/105124171/168682097-5db68455-bbeb-4e7c-8dd2-87a45638ced7.png)

**Comment by retorquere** [comment]
The thing is BBT should not have been involved. You have APA set as quick-copy format, why BBT cannot preload translators is something I've bee trying to figure out for a long time, but Zotero should not be calling up the BBT translator here at all. I'll have to wait and see what dstillman says on the forums.

**Comment by betamigo98** [comment]
Ok thanks for the info. Waiting for it I could just use the keyboard anyway.

**Comment by retorquere** [comment]
I've re-poked the issue on the forums, but if the devs aren't going to respond, there's nothing I can do.

**Comment by retorquere** [comment]
As far as I know, disabling for overlay plugins like bbt means "don't load at next start". If you disable bbt all functionality will be as it was until zotero shutdown.

**Comment by ReaderGuy42** [comment]
Yeah, sorry, I meant I did restart the program after each dis-/re-enable.

**Comment by 276562578** [comment]
Same issue, the annotation can't be ctrl+c or drag to other program after bbt enable and restart zotero. But it works after disable bbt and restart zotero.

**Comment by retorquere** [comment]
Still no response from the devs. Nothing I can do without that.

**Comment by retorquere** [comment]
I have re-prodded them on the [forums](https://forums.zotero.org/discussion/98245/dragging-articles-to-collections-is-slow/p1) and I have an open question on the [dev-forums](https://groups.google.com/g/zotero-dev/c/7UE3441sug8/m/U0PEuMPBAAAJ).
